vars/util.groovy

```diff
@@ -259,6 +259,16 @@ Integer[] parseVersion(String version) {
     }
 }
 
+String getMajorMinorVersion(String version) {
+    try {
+        String[] versionSplit = version.split("\\.")
```

Comment:
```suggestion
        String[] versionSplit = version.split('\\.')
```

Reply:
This is a tricky one so I won't change like that